arelle/plugin/validate/ESEF/Util.py

```diff
@@ -122,7 +125,9 @@ def resourcesFilePath(modelManager: ModelManager, fileName: str) -> str:
     return os.path.join(_resourcesDir, fileName)
 
 def loadAuthorityValidations(modelXbrl: ModelXbrl) -> list[Any] | dict[Any, Any]:
-    _file = openFileStream(modelXbrl.modelManager.cntlr, resourcesFilePath(modelXbrl.modelManager, "authority-validations.json"), 'rt', encoding='utf-8')  # type: ignore[no-untyped-call]
+    _file = openFileStream(modelXbrl.modelManager.cntlr, resourcesFilePath(modelXbrl.modelManager, "authority-validations.json"), 'rt', encoding='utf-8')
+    _file = cast("SupportsRead[Union[str, bytes]]", _file)
```

Comment:
This feels a bit hacky but `json.load` requires `SupportsRead[Union[str, bytes]]` and the `close()` method is only available on `io.IOBase` so we need to cast around here.

Reply:
Could you just do:

```python
with openFileString(...) as f:
   validations =  json.load(f)
```

instead? As a side note, I'm unsure what `openFileStream` is doing and why we couldn't just use `open`